app/experimenter/nimbus-ui/src/components/SidebarActions/index.test.tsx

```diff
@@ -235,4 +235,20 @@ describe("SidebarActions", () => {
       expect(screen.queryByTestId("CloneDialog")).not.toBeInTheDocument();
     });
   });
+
+  it("scrolls to json when preview recipe json button is clicked", async () => {
+    const experiment = mockExperiment({ isArchived: false, canArchive: true });
+    const refetch = jest.fn();
+    render(<Subject {...{ experiment, refetch }} />);
+
+    const jsonButton = await screen.findByTestId("button-recipe-json");
+
+    fireEvent.click(jsonButton);
+
+    await waitFor(() => {
+      expect(
+        screen.queryByTestId("button-recipe-json"),
+      ).not.toBeInTheDocument();
```

Comment:
Wait why is this the assertion?

Reply:
That's so weird, the fake test viewport has some invisible fixed size that makes this true for this particular layout?  That's so weird.  Okay well if it works for now let's just go with it.

Reply:
😂